src/core/server/saved_objects/migrationsv2/integration_tests/migration_7_13_0_unknown_types.test.ts

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import Path from 'path';
+import Fs from 'fs';
+import Util from 'util';
+import { estypes } from '@elastic/elasticsearch';
+import * as kbnTestServer from '../../../../test_helpers/kbn_server';
+import { Root } from '../../../root';
+
+const logFilePath = Path.join(__dirname, '7_13_unknown_types_test.log');
+
+const asyncUnlink = Util.promisify(Fs.unlink);
+
+async function removeLogFile() {
+  // ignore errors if it doesn't exist
+  await asyncUnlink(logFilePath).catch(() => void 0);
+}
+
+describe('migration v2', () => {
+  let esServer: kbnTestServer.TestElasticsearchUtils;
+  let root: Root;
+
+  beforeAll(async () => {
+    await removeLogFile();
+  });
+
+  afterAll(async () => {
+    if (root) {
+      await root.shutdown();
+    }
+    if (esServer) {
+      await esServer.stop();
+    }
+
+    await new Promise((resolve) => setTimeout(resolve, 10000));
+  });
+
+  it('migrates the documents to the highest version', async () => {
+    const { startES } = kbnTestServer.createTestServers({
+      adjustTimeout: (t: number) => jest.setTimeout(t),
+      settings: {
+        es: {
+          license: 'basic',
+          // dataset contains 2 type of unknown docs
+          // `foo` documents
+          // `space` documents (to mimic a migration with disabled plugins)
```

Comment:
Trying to make sure I understand why we don't need to set `xpack.spaces.enabled: false`. Is this because `createTestServers` only loads OSS plugins by default?

Reply:
Yea, it seems the test servers are only loading OSS by default (which made sense before the license changes)

Reply:
FWIW, I had to double check that again against what's included under the [default distribution](https://www.elastic.co/subscriptions). Maybe we should start referring to SSPL rather than OSS?